project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16
```

Comment:
I know in one of the SBT versions, `sbt test` stopped really running and testing the template itself. But I think this already happened in version 15

Reply:
Just tested and `sbt test` fails when I break something :)